test/utils.py

```diff
@@ -663,16 +663,16 @@ def get_command_line(client):
     return command_line
 
 
+def camel_to_upper_camel(camel):
+    return camel[0].upper() + camel[1:]
+
+
```

Comment:
Could you revert this change since it's not needed?

Reply:
Done